docsrc/Connecting_and_queries.rst

```diff
@@ -105,35 +105,35 @@ To get started, follow the steps below:
 
 **3. Execute commands using the cursor**
 
-	The ``cursor`` object can be used to send queries and commands to your Firebolt database and engine. See below for examples of functions using the ``cursor`` object. 
+	The ``cursor`` object can be used to send queries and commands to your Firebolt database and engine. See below for examples of functions using the ``cursor`` object.
 
-Command and query examples
+Server-side synchronous command and query examples
 ============================
 
-This section includes Python examples of various SQL commands and queries. 
+This section includes Python examples of various SQL commands and queries. For running queries in server-side asynchronous mode see :ref:`server-side asynchronous query execution`.
```

Comment:
What's the reasoning for adding this? Seems random to call out one feature of all the features listed in this section.

Reply:
Because server-side queries are a whole other section, not just one more feature, but I'm not wedded to this. I did mean to write something like "this is for server-side synchronous..."